packages/web/.storybook/preview-head.html

```diff
@@ -6,115 +6,42 @@
-  }
-
-  .css-pvgpy {
-    padding-block-start: 0 !important;
-  }
+  /* ----- PROPERTIES PAGE ----- */
 
   /* Component Preview */
-  .docs-story {
+  .docs-story,
+  .sb-show-main #storybook-root,
+  .docs-story .sb-story gcdscontainer .text-with-border {
     font-size: 0;
     color: var(--gcds-text-light);
+    padding: 0 !important;
+    border: none !important;
   }
 
-  /* Get code button */
-  .css-2w67r7 {
-    color: var(--gcds-text-light) !important;
-    font-size: 0 !important;
-    font-family: var(--gcds-font-families-body), sans-serif !important;
-    border-radius: var(--gcds-spacing-50) !important;
-    margin: var(--gcds-spacing-150) !important;
-  }
-
-  .css-2w67r7:not(:focus) {
-    border: var(--gcds-border-width-md) solid var(--gcds-color-grayscale-1000) !important;
-  }
-
-  .css-2w67r7:before {
-    font-size: .75rem;
-    color: var(--gcds-button-secondary-default-text) !important;
-    padding: var(--gcds-spacing-50);
-  }
-
-  .css-2w67r7:before {
-    content: 'Get code | Afficher le code';
-  }
-
-  .docblock-code-toggle--expanded.css-2w67r7:before {
-    content: 'Hide code | Masquer le code';
-  }
-
-  .css-2w67r7:focus {
-    background-color: var(--gcds-button-shared-focus-background) !important;
-    box-shadow: 0 0 0 var(--gcds-border-width-md) var(--gcds-button-shared-focus-text) !important;
-    outline: var(--gcds-spacing-50) solid var(--gcds-button-shared-focus-background) !important;
-    outline-offset: var(--gcds-border-width-md) !important;
-    color: var(--gcds-button-shared-focus-text) !important;
-  }
-
-  .css-2w67r7:focus:before {
-    color: var(--gcds-button-shared-focus-text) !important;
-  }
-
-  /* Copy code button */
-  .css-1az4em2 {
-    background-color: transparent !important;
-  }
-
-  .css-17vdvg6 {
-    color: rgba(0, 0, 0, 0.85) !important;
-    font-size: 0 !important;
-    border: var(--gcds-border-width-md) solid var(--gcds-color-grayscale-0) !important;
-    border-radius: var(--gcds-spacing-50) !important;
-    font-family: var(--gcds-font-families-body), sans-serif !important;
-    margin: var(--gcds-spacing-150) !important;
-  }
-
-  .css-17vdvg6:before {
-    content: 'Copy | Copie';
-    font-size: .75rem;
-    color: var(--gcds-text-light);
-    padding: var(--gcds-spacing-50);
-  }
-
-  .css-17vdvg6:focus {
-    background-color: var(--gcds-button-shared-focus-background) !important;
-    box-shadow: 0 0 0 var(--gcds-border-width-md) var(--gcds-button-shared-focus-text) !important;
-    outline: var(--gcds-spacing-50) solid var(--gcds-button-shared-focus-background) !important;
-    outline-offset: var(--gcds-border-width-md) !important;
-    color: var(--gcds-button-shared-focus-text) !important;
-  }
-
-  .css-17vdvg6:focus:before {
-    color: var(--gcds-button-shared-focus-text) !important;
+  .docs-story .hydrated,
+  .sb-show-main #storybook-root .hydrated {
+    font-size:var(--gcds-font-sizes-text) !important;
+    color: var(--gcds-text-primary) !important;
```

Comment:
It is both haha

We did decide to remove the red colour for labels and hint text for the error state to avoid overloading users with an ocean of red text in forms, but that being said, this styling was blocking some of our default styling and I updated it so that it doesn't happen anymore.

Added a note to my list to update the form error styling as well.